cpp/ps0002/ps0002_soln.cpp

```diff
@@ -4,23 +4,26 @@ using namespace std;
 
 int main()
 {
-    unsigned short int people = 28;//number 1
+    unsigned short int people = 28;//number 1 0.75/0.75
 
-    unsigned short int Alternative_medicine = 13 ;//number 2
+    unsigned short int Alternative_medicine = 13 ;//number 2 0.75/0.75
 
-    unsigned short int member = 7;//number 3
+    unsigned short int member = 7;//number 3 0.75/0.75
 
-    float bread = 4.5;//number 4
+    float bread = 4.5;//number 4 0.75/0.75
 
-    char first_letter = 'w';//number 5
+    char first_letter = 'w';//number 5 0.75/0.75
 
-    int Number_of_births = 9000000;//number 6
+    int Number_of_births = 9000000;//number 6 0.75/0.75
 
-    double divide_on_7 = 14.2857143;//number 7
+    double divide_on_7 = 14.2857143;//number 7 0.75/0.75
 
-    float Electricity_bill = 1400000;//number 8
+    float Electricity_bill = 1400000;//number 8 0.75/0.75
 
-    unsigned int age_in_scound = 10000000;//number 9
+    unsigned int age_in_scound = 10000000;//number 9 0.75/0.75
 
+    // total 6.75/6.75

```

Comment:
- [x] all variables required in the document are declared
- [ ] variables are declared with the correct data type
- [ ] using correct literals and notations for assigning initial values to the variables